client/public/index.html

```diff
@@ -12,6 +12,7 @@
   <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
   <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
   <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,700|Source+Code+Pro:400,700&amp;subset=latin-ext" rel="stylesheet">
+  <script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCAXMlEL-sfzT4jVK5CQHysSPp77JnVLks&libraries=places"></script>
```

Comment:
why this ? doesn't it exist as a package to install ?

Reply:
Unfortunately this needs to be loaded manually (https://github.com/kenny-hibino/react-places-autocomplete). But we can make the script load asynchronously so that it won't block the parser...